### DISCUSSION TITLE
Does the bot block those scammers from commenting on the channel ever again?

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#270] [Q&A] [answered]
I need some clarification on this.

**Comment by ThioJoe** [comment, marked as answer]
What do you think the ban option does 🤔

**Reply by Kurdakov** [reply]
from my experience those  bots, which were held for review but not were banned (the channel owner is a traveler, so has few time to check for details) next time are shadow banned - their comments appear, but then quickly disappear (though, possibly there was another moderator working - though it's unlikely moderators known to me were not active at a time)

**Reply by KendallDoesCoding** [reply]
> What do you think the ban option does 🤔

Sorry for the stupid question. I haven't used the program in a while lol
